include/envoy/upstream/upstream.h

```diff
@@ -611,16 +611,16 @@ class PrioritySet {
  * can be handled differently by passing in different macros.
  */
 #define ALL_CLUSTER_CIRCUIT_BREAKERS_STATS(OPEN_GAUGE, REMAINING_GAUGE)                            \
-  OPEN_GAUGE(cx_open, Accumulate)                                                                  \
-  OPEN_GAUGE(cx_pool_open, Accumulate)                                                             \
-  OPEN_GAUGE(rq_open, Accumulate)                                                                  \
-  OPEN_GAUGE(rq_pending_open, Accumulate)                                                          \
-  OPEN_GAUGE(rq_retry_open, Accumulate)                                                            \
-  REMAINING_GAUGE(remaining_cx, Accumulate)                                                        \
-  REMAINING_GAUGE(remaining_cx_pools, Accumulate)                                                  \
-  REMAINING_GAUGE(remaining_pending, Accumulate)                                                   \
-  REMAINING_GAUGE(remaining_retries, Accumulate)                                                   \
-  REMAINING_GAUGE(remaining_rq, Accumulate)
+  OPEN_GAUGE(cx_open, NeverImport)                                                                 \
+  OPEN_GAUGE(cx_pool_open, NeverImport)                                                            \
+  OPEN_GAUGE(rq_open, NeverImport)                                                                 \
+  OPEN_GAUGE(rq_pending_open, NeverImport)                                                         \
+  OPEN_GAUGE(rq_retry_open, NeverImport)                                                           \
+  REMAINING_GAUGE(remaining_cx, NeverImport)                                                       \
+  REMAINING_GAUGE(remaining_cx_pools, NeverImport)                                                 \
+  REMAINING_GAUGE(remaining_pending, NeverImport)                                                  \
+  REMAINING_GAUGE(remaining_retries, NeverImport)                                                  \
+  REMAINING_GAUGE(remaining_rq, NeverImport)
```

Comment:
I think these make sense as NeverImport, though there may be some loss of fidelity here that I think is worth some doc updates. @tonya11en wdyt?

Reply:
Unless someone increased the global request timeout, it should only be inaccurate for ~15 seconds in the absolute worst case. +1 on the doc updates.